src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.ts

```diff
@@ -118,16 +117,6 @@ export class TelemetryComponent extends CdForm implements OnInit {
   private getValidators(option: any): ValidatorFn[] {
     const result = [];
     switch (option.type) {
-      case 'int':
-        result.push(CdValidators.number());
-        result.push(Validators.required);
-        if (_.isNumber(option.min)) {
-          result.push(Validators.min(option.min));
-        }
-        if (_.isNumber(option.max)) {
-          result.push(Validators.max(option.max));
-        }
-        break;
```

Comment:
this is an incorrect removal, i'll have to take care of it. so I'll put DNM label and will take care of this once I am back